### PR TITLE
Added Lenovo Thinkpad X390

### PR DESCRIPTION
**Pull request by paul-jewell**
###### Description of changes
Added Lenovo Thinkpad X390 model

###### Things done

- [ x ] Tested the changes in your own NixOS Configuration
- [ x ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

File: x390 [hunk 1]
```diff
@@ -0,0 +1,45 @@
+diff --git a/README.md b/README.md
```
Comment (Mic92) [inline x390:1]: Looks like you added this by accident?

Comment (Mic92) [reply x390:1]: looks good otherwise

Comment (paul-jewell) [reply x390:1]: Oops - I will clean this up - thanks for pointing it out.

Comment (paul-jewell) [reply x390]: I have removed the offending section, but for some reason I can't fathom, there is now the last commit from master showing in my pull request, even though it isn't in my commit. Please accept my apologies - I am not sure how I have managed to do this, and apart from deleting everything and starting again, I am not sure how to fix it. Before I do this, I would appreciate any tips you may be able to give me reference fixing this!

Comment (paul-jewell) [reply x390]: I have now removed the offending file!

**Comment by Mic92** [comment]
bors merge

**Pull request merged**